res/values-es/strings.xml

```diff
@@ -271,7 +271,7 @@
 
     <string name="CREATE_CLAN">"CREAR CLAN"</string>
     <string name="MEMBER">"MIEMBRO"</string>
-    <string name="ADMIN">"ADMIN"</string>
+    <string name="ADMIN">"ADMINISTRADOR"</string>
```

Comment:
Please return as "ADMIN" here, because it's already understandable by spanish-speakers and length cannot be too long.